### PR TITLE
Updated to work with Visual Recognition v3

### PR DESCRIPTION
**Pull request by purukn**

### CODE REVIEW

File: INSTRUCTIONS.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Leveraging the IBM Bluemix Devops Services, we are able to quickly clone the cod
 
   1. Click the button below to fork the project into IBM DevOps Services and deploy your own instance of this application on [IBM Bluemix][bluemix].
 
-  [![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/watson-developer-cloud/image-analysis)
+  [![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/purukn/image-analysis)
```
Comment (kognate) [inline INSTRUCTIONS.md]: This should be changed back to watson-developer-cloud.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Lab Instructions: [INSTRUCTIONS](INSTRUCTIONS.md)
 Demo: [http://image-analysis.mybluemix.net/](http://image-analysis.mybluemix.net/)
 
 
-[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/watson-developer-cloud/image-analysis)
+[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/purukn/image-analysis)
```
Comment (kognate) [inline README.md]: This should be changed back to watson-developer-cloud

File: routes/vr.js [hunk 3]
```diff
@@ -40,9 +39,13 @@ module.exports.recognize = function(req, res, next) {
     if (req.file)
       fs.unlink(req.file.path);
 
-		if (error)
+		if (error) {
+			console.log(error);
```
Comment (kognate) [inline routes/vr.js]: These debug log statements should be removed from released code.

**Comment by purukn** [comment]
Corrected the links and removed comments in the latest commit

File: README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ Lab Instructions: [INSTRUCTIONS](INSTRUCTIONS.md)
 Demo: [http://image-analysis.mybluemix.net/](http://image-analysis.mybluemix.net/)
 
 
-[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/watson-developer-cloud/image-analysis)
+[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/wwatson-developer-cloud/image-analysis)
```
Comment (kognate) [inline README.md]: Watson only has one w character.

Comment (purukn) [inline README.md]: corrected the URL

**Pull request merged**